cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -130,17 +115,23 @@ def test_provider_summary_scvmm():
     """
     The purpose of this test is to verify that the information on the
     provider summary is substantially the same as what is on SCVMM.
-    Since SCVMM-SP1 only has a short sequence of test cases, you must use
-    this test case as the catch all to go in and spend 15-30 minutes and
-    check as many links from this page and verify both the navigation and
-    the content.
+    Note: when automated this test will likely be broken up into many tests.
+
 
     Polarion:
         assignee: jdupuy
         casecomponent: Infra
         caseimportance: medium
         initialEstimate: 1/2h
-        startsin: 5.4
+        startsin: 5.7
+        testSteps:
```

Comment:
Question(Just curious):
- Why did you change ```startsin``` version?

Reply:
Hmm... actually not sure about that one, probably to match the other startsin? I will leave it as 5.4